requirements.txt

```diff
@@ -0,0 +1 @@
+coverage==7.1.0
```

Comment:
Use poetry for dependencies.

Reply:
deleted req.txt and added coverage to poetry